### DISCUSSION TITLE
Information on CRM

### DISCUSSION THREAD

**Discussion by informaticaopensource** [YetiForceCompany/YetiForceCRM#15636] [General]
Hello everyone;
I need to understand some questions about using CRM.
In particular:
1- Can I customize the template and remove the lower banner?
2- The second question is related to the first. If my company wants to resell the product, what conditions are required for you?
3- The last question is more technical; can I disable some fields at the start of the worfklow and enable and disable them when necessary?
Thanks for all the answers.

**Comment by bpabiszczak** [comment]
1. "lower banner" it is added automatically, you need to purchase this add-on for it to disappear: https://yetiforce.com/pl/yetiforce-branding
2. To be able to sell our addons, you must become a partner, if you want your company on the invoice, you must have this partnership: https://yetiforce.com/en/integrator-partnership
3. Check the available panels here:  Software configuration > Standard modules > Dependent Fields

**Reply by informaticaopensource** [reply]
Hi, thank.
2- If our company were interested in becoming a partner, would it be possible to communicate in Italian by telephone at your contact details?

**Comment by bpabiszczak** [comment]
only pl/en.
